torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -302,6 +302,29 @@ at::Tensor AtenXlaType::_adaptive_avg_pool2d_backward(
       bridge::GetXlaTensor(grad_output), bridge::GetXlaTensor(self)));
 }
 
+void AtenXlaType::_amp_foreach_non_finite_check_and_unscale_(
+    at::TensorList self, at::Tensor& found_inf, const at::Tensor& inv_scale) {
+  XLA_FN_COUNTER("xla::");
+  auto xla_self = bridge::GetXlaTensors(self);
+  absl::Span<XLATensor> self_tensor{xla_self};
+  XLATensor found_inf_tensor = bridge::GetXlaTensor(found_inf);
```

Comment:
Done. But some of the tensors should not use `bridge::GetXlaTensor(found_inf)` directly since it needs to be modified inplace and should not be a temporary variable.